### ISSUE TITLE
Add static bool32 members

### ISSUE THREAD

**Issue by realvictorprm** [FacticiusVir/SharpVk#35]
Hi,
during my work with SharpVK I noticed that I need to apply every time casts to my bool values etc.

Instead of conversions I would like to see additional static members e.g.
``` csharp
partial class Bool32 {
   public static bool True = Bool32(true);
   public static bool False = Bool32(false);
}
```

**Comment by FacticiusVir** [comment]
Oh, that's unpleasant; there are implicit casts between bool and Bool32, but if you're using F# those aren't compatible. I'll see about removing Bool32 from the public API, if that's not workable I'll add some Bool32.True/False shortcuts.

**Comment by realvictorprm** [comment]
This would be nice :)

**Comment by FacticiusVir** [comment]
The resolution for this issue is included in release 0.4.0; Bool32 should be automatically marshalled from bool in all uses in the public API.
